src/index.js

```diff
@@ -4,5 +4,7 @@ module.exports = {
 	healthchecks: {
 		CanQueryList: require('./healthchecks/CanQueryList'),
 		canQueryListFactory: require('./healthchecks/canQueryListFactory'),
+		CanQueryUri: require('./healthchecks/CanQueryUri'),
```

Comment:
this will fail on any OS that isn't MacOS. The case of the filename is different to the case of the string?